ProfITsoftTestTask/Controller/NewWorkerViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  NewWorkerViewController.swift
+//  ProfITsoftTestTask
+//
+//  Created by Nikita Kalyuzhnyy on 01.02.2021.
+//
+
+import Foundation
+import UIKit
+import CoreData
+
+class NewWorkerViewController: UITableViewController {
+    
+    var selectedCompany: String = "None"
+    
+    @IBAction func saveButton(_ sender: Any) {
+        alertMessage()
+        saveWorker()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        tableView.reloadData()
+    }
+    
+    override func viewDidLoad() {
+        
+        tableView.dataSource = self
+        tableView.delegate = self
+    }
+    
+    private func alertMessage() {
+        
+        let alertController = UIAlertController(title: "", message: "Saving done ✅", preferredStyle: .alert)
+        
+        let cancelAction = UIAlertAction(title: "OK", style: .cancel) { (action:UIAlertAction) in
+            self.navigationController?.popViewController(animated: true)
+        }
+                
+        alertController.addAction(cancelAction)
+        self.present(alertController, animated: true, completion: nil)
+    }
+    
+    private func saveWorker() {
+        
+        let workersDictionary: [String: Any] = [
+        
+            "imageURL": ImageCell.imageFromImageCell,
```

Comment:
1. Create model to store imageURL, name,lastName, etc
2. Pass model to DataManager